apigw-lambda-bedrock-sam-java/example-pattern.json

```diff
@@ -0,0 +1,61 @@
+{
+  "title": "Invoking Amazon Bedrock Mistral 7B LLM using Java based AWS Lambda",
+  "description": "This project deploys an Amazon API Gateway REST API with an AWS Lambda, written in Java, to call Amazon Bedrock API for Mistral 7B Instruct V0.2.",
+  "language": "Java",
+  "level": "200",
+  "framework": "SAM",
+  "introBox": {
+    "headline": "How it works",
+    "text": [
+      "Amazon API Gateway receives the HTTP POST request containing the prompt.",
+      "AWS Lambda is triggered by the API Gateway, this function forwards the prompt to Amazon Bedrock Java API. It uses Mistral 7B Instruct V0.2 model and sets other required parameters to fixed values for simplicity.",
+      "Based on the given prompt, using Mistral 7B Instruct V0.2 model, Amazon Bedrock generates the content and returns the response to the Lambda function.",
+      "The Lambda function processes the Bedrock API response and sends it back to the user via the API Gateway."
+    ]
+  },
+  "gitHub": {
+    "template": {
+      "repoURL": "https://github.com/aws-samples/serverless-patterns/tree/main/apigw-lambda-bedrock-sam-java",
+      "templateURL": "serverless-patterns/apigw-lambda-bedrock-sam-java",
+      "projectFolder": "apigw-lambda-bedrock-sam-java",
+      "templateFile": "apigw-lambda-bedrock-sam-java/template.yaml"
```

Comment:
Please remove the projectFolder value from the templateFile

Reply:
Fixed.